Applications/Scale/test/testScale.cpp

```diff
@@ -177,7 +185,13 @@ void scaleGait2354_GUI(bool useMarkerPlacement)
 
     // Compare ScaleSet
     ScaleSet stdScaleSet = ScaleSet(setupFilePath+"std_subject01_scaleSet_applied.xml");
-
+    // We used to compute a scale factor for the ground body, which does not make physical sense.
+    // Things connected to ground can be scaled, including their locations, but not the frame itself.
+    // Now that ground is its own frame it does not go through the body scale mill.
+    // We must remove it from the stdScaleSet.
```

Comment:
Maybe all frames, including ground, should have scale methods. If I have a frame/body called the Femur, then I attach muscles and joints to the Femur, the attachment locations of all frames, muscles, and other components should increase and decrease with the scale factor of the Femur.  I see that being useful for ground as well.  Scaling ground would increase or decrease the attachment vectors of everything attached to it.

With that I think ground having scale methods makes sense.  Perhaps these should just be operators that morph attachements of all connected frames and such, but don't preserve a "scale factor" for the ground.  I totally understand that a ground scale factor is strange, but the same scale methods would be useful.